src/vstest.console/Processors/TestCaseFilterArgumentProcessor.cs

```diff
@@ -1,4 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
It looks like you've duplicated the comment. Please remove the redundant line :)